lib/themes/dosomething/paraneue_dosomething/js/components/carousel.js

```diff
@@ -0,0 +1,30 @@
+const React = require('react');
+
+import ContextSlide from './context_slide.js';
+
+const Carousel = React.createClass({
```

Comment:
Let's use ES6 classes for our stateful components. The `React.createClass` syntax is more-or-less deprecated:

```js
class Carousel extends React.Component {
   constructor(props) {
     super(props);

     this.state = { activeSlide : 1 };
   }

   render() {
     // ...
   }
}